sklearn/preprocessing/_encoders.py

```diff
@@ -171,6 +171,8 @@ class OneHotEncoder(_BaseEncoder):
 
     Read more in the :ref:`User Guide <preprocessing_categorical_features>`.
 
+    .. versionadded:: 0.13
+
```

Comment:
this one actually would deserve a versionchanged if you like with the change from 0.20:
https://scikit-learn.org/dev/whats_new/v0.20.html#id37

but writing this well might actually requiring understanding what in the world went on there and it's a bit tricky ;)

Reply:
No worries, I will switch to versionchanged. Indeed, it may be hard to judge what was a major change.

Reply:
@amueller made the change as you suggested :)